[2024-05-25T08:50:06.719+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-25T08:50:06.742+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v03.transform_data_response manual__2024-05-25T08:13:51.839954+00:00 [queued]>
[2024-05-25T08:50:06.752+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v03.transform_data_response manual__2024-05-25T08:13:51.839954+00:00 [queued]>
[2024-05-25T08:50:06.753+0000] {taskinstance.py:2306} INFO - Starting attempt 9 of 14
[2024-05-25T08:50:06.767+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data_response> on 2024-05-25 08:13:51.839954+00:00
[2024-05-25T08:50:06.775+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_weather_workflow_DAG_v03', 'transform_data_response', 'manual__2024-05-25T08:13:51.839954+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmppw3v7k8f']
[2024-05-25T08:50:06.778+0000] {standard_task_runner.py:91} INFO - Job 104: Subtask transform_data_response
[2024-05-25T08:50:06.779+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=5527) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-25T08:50:06.781+0000] {standard_task_runner.py:63} INFO - Started process 5529 to run task
[2024-05-25T08:50:06.845+0000] {task_command.py:426} INFO - Running <TaskInstance: my_weather_workflow_DAG_v03.transform_data_response manual__2024-05-25T08:13:51.839954+00:00 [running]> on host 6ab40057742d
[2024-05-25T08:50:07.120+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Khang' AIRFLOW_CTX_DAG_ID='my_weather_workflow_DAG_v03' AIRFLOW_CTX_TASK_ID='transform_data_response' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T08:13:51.839954+00:00' AIRFLOW_CTX_TRY_NUMBER='9' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-25T08:13:51.839954+00:00'
[2024-05-25T08:50:07.122+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-25T08:50:07.143+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-25T08:50:07.144+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-25T08:50:07.160+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_weather_workflow_DAG_v03, task_id=transform_data_response, run_id=manual__2024-05-25T08:13:51.839954+00:00, execution_date=20240525T081351, start_date=20240525T085006, end_date=20240525T085007
[2024-05-25T08:50:07.197+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-25T08:50:07.221+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-25T08:50:07.224+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
